migrations/20231210123554_init.sql

```diff
@@ -0,0 +1,314 @@
+    FOREIGN KEY  (event_id) REFERENCES event (id),
+    --------------------------------------------------------
+    CONSTRAINT check_timesheet_start_date_lte_end_date
+        CHECK (start_date >= end_date),
+    CONSTRAINT check_timesheet_created_at_lte_edited_at
+        CHECK (edited_at >= created_at)
+);
+
+
+CREATE TABLE work_day
+(   
+    date         DATE NOT NULL,
+    total_hours  hours_per_day_float NOT NULL,
+    comment      TEXT,
+    is_editable  BOOLEAN NOT NULL,
+    --------------------------------------------------------
+    created_at   TIMESTAMP NOT NULL DEFAULT now(),
+    edited_at    TIMESTAMP NOT NULL DEFAULT now(),
+    deleted_at   TIMESTAMP,
+    --------------------------------------------------------
+    timesheet_id UUID,
+    --------------------------------------------------------
+    FOREIGN KEY  (timesheet_id) REFERENCES timesheet (id),
+    --------------------------------------------------------
+    CONSTRAINT check_work_day_created_at_lte_edited_at
+        CHECK (edited_at >= created_at)
+);
+
+
+CREATE TABLE event_staff
+(
+    id          UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+    -------------------------------------------------------
+    level       employee_level NOT NULL,
+    -------------------------------------------------------
+    created_at  TIMESTAMP NOT NULL DEFAULT now(),
+    edited_at   TIMESTAMP NOT NULL DEFAULT now(),
+    deleted_at  TIMESTAMP,
+    -------------------------------------------------------
+    user_id     UUID,
+    event_id    UUID,
+    -------------------------------------------------------
+    FOREIGN KEY (user_id) REFERENCES "user" (id),
+    FOREIGN KEY (event_id) REFERENCES event (id),
+    -------------------------------------------------------
+    CONSTRAINT check_event_stuff_created_at_lte_edited_at
+        CHECK (edited_at >= created_at)
+);
+
+
+CREATE TABLE task
+(
+    id          UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+    -------------------------------------------------------
+    title       VARCHAR(255) NOT NULL,
+    description TEXT,
+    finished_at DATE,
+    priority    task_priority NOT NULL,
+    accepted    acceptance_status NOT NULL,
+    -------------------------------------------------------
+    created_at  TIMESTAMP NOT NULL DEFAULT now(),
+    edited_at   TIMESTAMP NOT NULL DEFAULT now(),
+    deleted_at  TIMESTAMP,
+    -------------------------------------------------------
+    event_id    UUID,
+    creator_id  UUID,
+    -------------------------------------------------------
+    FOREIGN KEY (event_id) REFERENCES event (id),
+    FOREIGN KEY (creator_id) REFERENCES event_staff (id),
+    -------------------------------------------------------
+    CONSTRAINT check_event_title_len
+        CHECK (char_length(title) >= 1),
+    CONSTRAINT check_event_stuff_created_at_lte_edited_at
+        CHECK (edited_at >= created_at)
+);
+
+
+CREATE TABLE assigned_staff
+(
+    status      acceptance_status NOT NULL,
```

Comment:
This table is missing the decided_by: <> (FK event_staff) column.
The decided_by column is connected to the acceptance_status column, and informs the worker of which responsible person made the decision on their participation in the handling of a task.